packages/components/src/internal/components/assay/AssayPicker.spec.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
 import { mount } from 'enzyme';
 
-import { Simulate } from 'react-dom/test-utils';
+import { AssayPicker, AssayPickerTabs, sleep } from '../../..';
+import { initMocks } from '../../../stories/mock';
 
-import { AssayPicker, AssayPickerTabs } from '../../..';
+beforeAll(() => {
+    initMocks();
```

Comment:
This should not be using the storybook mocks but rather the unit test mocks.

```ts
import { initUnitTestMocks } from '../../testHelperMocks';

beforeAll(() => {
    initUnitTestMocks();
});
```

Reply:
Seems like we reached a performance threshold or something with this function, where adding more mock apis was causing mock api issues in other tests.  I updated this function to only include some more widely used mocks and added a param to pass in test specific mocks.